lib/manager/actions/status.js

```diff
@@ -137,44 +138,73 @@ export async function fetchAppInfo () {
   }
 }
 
+/**
+ * Constructs modal content from an service period merge (principally used for
+ * MTC).
+ */
 function getMergeFeedModalContent (result: MergeFeedsResult): ModalContent {
+  const SEPARATOR = ': '
+  const {
+    errorCount,
+    failed,
+    failureReasons,
+    idConflicts,
+    mergeStrategy,
+    remappedIds,
+    remappedReferences,
+    skippedIds
+  } = result
+  const messages = getComponentMessages('MergeFeedsResult')
   const details = []
+  details.push(
+    [messages('strategyUsed'), messages(mergeStrategy)].join(SEPARATOR)
+  )
   // Do nothing or show merged feed modal? Feed version is be created
-  details.push('Remapped ID count: ' + result.remappedReferences)
-  if (Object.keys(result.remappedIds).length > 0) {
+  details.push([messages('remappedIdCount'), remappedReferences].join(SEPARATOR))
+  if (Object.keys(remappedIds).length > 0) {
     const remappedIdStrings = []
-    for (let key in result.remappedIds) {
+    for (const key in remappedIds) {
       // Modify key to remove feed name.
       const split = key.split(':')
       const tableAndId = split.splice(1, 1)
-      remappedIdStrings.push(`${tableAndId.join(':')} -> ${result.remappedIds[key]}`)
+      remappedIdStrings.push(`${tableAndId.join(':')} -> ${remappedIds[key]}`)
     }
-    details.push('Remapped IDs: ' + remappedIdStrings.join(', '))
+    details.push(
+      [messages('remappedIds'), remappedIdStrings.join(', ')].join(SEPARATOR)
```

Comment:
Could there be a template string used in the english.yml file instead?

Reply:
Improved in https://github.com/ibi-group/datatools-ui/pull/686/commits/1c1239097f50b08c9c2172d9de671f91ae96cc77 with a proper list.